components/ciphertool/src/main/java/org/wso2/ciphertool/cipher/SymmetricCipher.java

```diff
@@ -66,13 +63,13 @@ public SymmetricCipher(KeyStore keyStore, String keyAlias) {
         try {
             this.secretKey = keyStore.getKey(keyAlias, password.toCharArray());
             if (this.secretKey == null) {
-                throw new KeyStoreException(GET_KEY_ERROR_MESSAGE + keyAlias);
+                throw new KeyStoreException(Constants.Error.GET_KEY_ERROR_MESSAGE + keyAlias);
```

Comment:
Enum property getter not called and keyAlias is concatenated instead of formatted in the enum property message. Fix in line 72 as well.